### ISSUE TITLE
Rule angular/deferred doesn't raise an error on _$q_.defer()

### ISSUE THREAD

**Issue by ThomasvLingen** [Gillespie59/eslint-plugin-angular#325]
Came across this whilst working on something and wasn't completely sure if this was intentional.

For example, the deferred rule is totally alright with this:
```javascript
var deferred;
    
beforeEach(inject(function(_$q_) {
    deferred = _$q_.defer();
}));
```

**Comment by Gillespie59** [comment]
Yep, because we only test `$q` and not `_$q_` us in test. 
We solve that.

**Comment by ThomasvLingen** [comment]
Awesome! Thanks a bunch!
